crates/humanode-runtime/src/tests/claims_and_vesting.rs

```diff
@@ -176,6 +182,21 @@ fn new_test_ext() -> sp_io::TestExternalities {
             ],
             total_claimable: Some(2 * VESTING_BALANCE),
         },
+        evm: EVMConfig {
+            accounts: {
+                let evm_pot_accounts = vec![(
+                    EvmToNativeSwapBridgePot::account_id(),
+                    fp_evm::GenesisAccount {
+                        balance: INIT_BALANCE.into(),
+                        code: Default::default(),
+                        nonce: Default::default(),
+                        storage: Default::default(),
+                    },
+                )];
+
+                evm_pot_accounts.into_iter().collect::<BTreeMap<_, _>>()
```

Comment:
```suggestion
                evm_pot_accounts.into_iter().collect()
```

Can we avoid mentioning the `BTreeMap` altogether here?

Reply:
Thanks. It works - https://github.com/humanode-network/humanode/pull/673/commits/557cdcdfab77c3e1b531fc9037bcf81e459ee5c5